editor.planx.uk/src/@planx/components/PlanningConstraints/List.tsx

```diff
@@ -123,100 +132,71 @@ interface ConstraintListItemProps {
 }
 
 function ConstraintListItem({ children, ...props }: ConstraintListItemProps) {
-  const [showConstraintData, setShowConstraintData] = useState<boolean>(false);
 
   return (
     <ListItem disablePadding sx={{ backgroundColor: "white" }}>
-      <StyledConstraint {...props}>
-        <Box
-          style={{
-            display: "flex",
-            flexDirection: "row",
-            justifyContent: "space-between",
-          }}
+      <StyledAccordion {...props} disableGutters>
+        <AccordionSummary
+          id={`${props.content}-header`}
```

Comment:
This is producing an ID with spaces which will result in multiple IDs per element, and replicated IDs:
`id="is on a Classified Road-header"`